src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagerGetTaskConsumerTest.java

```diff
@@ -162,6 +192,19 @@ private DslPart createResponseForGetTask() {
                 )).build();
     }
 
+
+    private DslPart createResponseForGetTaskWithCompletionProcess() {
+        return LambdaDsl.newJsonBody(o -> o.object("task", task -> {
+            createResponseForGetTask();
+            task.stringMatcher(
+                "termination_process",
+                "EXUI_USER_COMPLETION|EXUI_CASE-EVENT_COMPLETION",
+                "EXUI_USER_COMPLETION"
```

Comment:
similar comment to the one above about the search.  A Task may not have a `termination_process` field.